fake.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2015 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package testing
+
+import (
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+)
+
+// FakeCallArgs holds all the args passed in a call.
+type FakeCallArgs map[string]interface{}
```

Comment:
I did it to help in couple spots, like when calling CheckCalls().  `testing.FakeCallArgs{...}` is nicer to use (and read) than `map[string]interface{}{...}`.  There are a number of other places in juju where we do the same thing (e.g. testing.Attrs).